core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -2884,6 +2879,9 @@ public void run() {
     public Set<Transaction> getTransactions(boolean includeDead) {
         lock.lock();
         try {
+            if (includeDead) {
```

Comment:
`includeDead` can be removed further down this method. Else, probably a valid improvement.